ServiceBusDashboard/Controllers/DashboardController.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Web;
+using System.Web.Http;
+using Microsoft.ServiceBus;
+using ServiceBusDashboard.Code;
+using WebGrease.Css.Extensions;
+
+namespace ServiceBusDashboard.Controllers
+{
+    [RoutePrefix("dashboard")]
+    public class DashboardController : ApiController
+    {
+        [HttpGet]
+        [Route("{serviceBusName}")]
+        public IHttpActionResult Get(string serviceBusName)
+        {
+            const string ignoreQueues = "";
+            const string ignoreTopics = "";
+            const string ignoreSubscriptions = "^test(.*)$,^example(.*)$";
+            //const string replaceTheseStartWordsWithDots = "inputqueue.interxion.";
+            const string replaceTheseStartWordsWithDots = "";
```

Comment:
What is this used for? Perhaps a different variable name plus a comment on its intent?

Reply:
I will remove using "replaceTheseStartWordsWithDots" as it makes the code more complicated and is not much usable.